torch/_decomp/decompositions_for_jvp.py

```diff
@@ -145,7 +195,9 @@ def prod(x: List[int]):
     return r
 
 
-@register_decomposition_for_jvp(aten.native_batch_norm_backward)  # @register_decomposition_for_jvp after in core
+@register_decomposition_for_jvp(
+    aten.native_batch_norm_backward
+)  # @register_decomposition_for_jvp after in core
```

Comment:
Super nit but since we're in here, could we remove this comment? This should have been removed earlier by us